doc/INSTALL.md

```diff
@@ -66,7 +66,6 @@ Add the path where the library has been installed
 
 and run `ldconfig` as root.
 
-
```

Comment:
Yes, the "Installation Manuals"  section should be in doc/tutorials/README.md , but in INSTALL.md we could mention that some specific installations setups are described in doc/tutorials/README.md?